pkgs/tools/typesetting/sile/default.nix

```diff
@@ -42,56 +44,69 @@ let
   ] ++ lib.optionals (lib.versionOlder lua.luaversion "5.3") [
     compat53
   ]);
-in
 
-stdenv.mkDerivation (finalAttrs: {
+  in stdenv.mkDerivation (finalAttrs: {
   pname = "sile";
-  version = "0.14.17";
+  version = "0.15.4";
 
   src = fetchurl {
-    url = "https://github.com/sile-typesetter/sile/releases/download/v${finalAttrs.version}/sile-${finalAttrs.version}.tar.xz";
-    sha256 = "sha256-f4m+3s7au1FoJQrZ3YDAntKJyOiMPQ11bS0dku4GXgQ=";
+    url = "https://github.com/sile-typesetter/sile/releases/download/v${finalAttrs.version}/sile-${finalAttrs.version}.tar.zst";
+    sha256 = "0n13kv1pabqccjycswcfafhks54l9ck6k8dfkd436jggharkgn1m";
   };
 
-  configureFlags = [
-    "--with-system-luarocks"
-    "--with-manual"
-  ];
-
   nativeBuildInputs = [
-    gitMinimal
-    pkg-config
     makeWrapper
+    pkg-config
+    rustPlatform.cargoSetupHook
+    jq
+    cargo
+    rustc
+    zstd
```

Comment:
```suggestion
```

Reply:
For some reason this did not work for me, taking it out of here and adding it into the fetchurl hunk above made the build start erroring out about not being able to execute zstd.